docs/UserGuide.md

```diff
@@ -85,8 +85,8 @@ A person can have any number of tags (including 0)
 </div>
 
```

Comment:
Format needs to update, and `MESSAGE_USAGE` under `AddCommand.java`

Reply:
Updated